frontend/package.json

```diff
@@ -15,6 +15,7 @@
     "Instead update 'web_root' in Kaili config map in OpenShift console."
   ],
   "homepage": "./",
+  "proxy": "http://localhost:8000",
```

Comment:
Is this from testing?
```suggestion
```

Reply:
Och, sorry, I thought it was fixed.
Thanks